cmd/lncli/neutrino_active.go

```diff
@@ -225,6 +225,51 @@ func getCFilter(ctx *cli.Context) error {
 	return nil
 }
 
+var unbanPeerCommand = cli.Command{
+	Name:        "unbanPeer",
+	Usage:       "Unbans a previously banned peer and connects to it.",
+	Category:    "Neutrino",
+	Description: "Unban and connects to a previously banned peer",
+	ArgsUsage:   "address",
+	Action:      actionDecorator(unbanPeer),
+	Flags: []cli.Flag{
+		cli.BoolFlag{
+			Name: "permanent",
+			Usage: "indicates if the peer to be unbanned should " +
+				"be connected as a permanent peer. If set, " +
+				"it connects as a permanent peer",
+		},
+	},
+}
+
+func unbanPeer(ctx *cli.Context) error {
+	ctxc := getContext()
+	args := ctx.Args()
+
+	// Display the command's help message if we do not have the expected
+	// number of arguments/flags.
+	if ctx.NArg() != 1 || ctx.NumFlags() > 1 {
+		return cli.ShowCommandHelp(ctx, "unbanpeer")
```

Comment:
this doesnt work cause it doesnt match the command name. Suggest changing the command name to this though

Reply:
Nice catch, thanks!